ombi/ombi-init-update.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Script Name: AtoMiC Ombi init.d update
+# Author: mgiljum
+# Publisher: http://www.htpcbeginner.com
+#
+
+# DO NOT EDIT ANYTHING UNLESS YOU KNOW WHAT YOU ARE DOING.
+echo 'Updating init.d file'
+
+sudo sed -i "s@DAEMON_USER=ReplaceMe@RUNASUSER=$UNAME@g" /etc/init.d/$APPNAME || { echo -e $RED'Replacing RUN_AS failed.'$ENDCOLOR ; exit 1; }
```

Comment:
DAEMON_USER=\"ReplaceMe\"@DAEMON_USER=\"$UNAME\"@

it also needs a \ before " on the 4 " (see the deluged default update as an example) i can't seem to get it to show up on the comment!